UNRELEASED_TOKENS.md

```diff
@@ -0,0 +1,52 @@
+| --------------- | -------------------------------------------------------------------------- |
+| HoprDistributor | a live smart contract on **mainnet** that users can claim HOPR tokens      |
+| CLAIMANT_ADDR   | an **ethereum address** that can claim HOPR tokens through HoprDistributor |
+| HOPR_ID         | a **HOPR id** of a HOPR node                                               |
+
+## Process
+
+1. User sends an email to tech@hoprnet.org with subject `Link my HOPR node`, and a message containing their `HOPR_ID` and the signature of `Link my Ethereum address to HOPR node address <HOPR_ID>` that was signed using `CLAIMANT_ADDR`. Example:
+
+```
+16Uiu2HAmQBZA4TzjKjU5fpCSprGuM2y8mpepNwMS6ZKFATiKg68h
+<signature of "Link my Ethereum address to HOPR node address 16Uiu2HAmQBZA4TzjKjU5fpCSprGuM2y8mpepNwMS6ZKFATiKg68h>
+```
+
+2. PM creates an issue containing user's email and assigns a [team member](#who-is-team-member) to tackle.
+3. Team member must then verify the validity of the request:
+   1. Signer address (aka `CLAIMANT_ADDR`) of the signature has claimable tokens on schedules `EarlyTokenBuyers` or `TeamAndAdvisors` _(link to etherscan method)_.
+   2. Signer does not already exist in [unreleasedTokens.json](./packages/cover-traffic/unreleasedTokens.json)
+   3. Inserted `HOPR_ID` is a valid HOPR ID _(link to tooling)_
+4. Team member adds another entry to [unreleasedTokens.json](./packages/cover-traffic/unreleasedTokens.json) using syntax `{ tokens: string, ethAddress: string, hoprId: string }[]`
+5. Team member creates a PR so another team member can review and approve, from here on we follow our usual [WORKFLOW](./WORKFLOW.md) steps.
+6. Once merged, team member replies to user with:
+
+```
+Hello,
+
+Your HOPR ID has been added into the list.
+```
+
+## Who is "team member"
+
+The release's [PM](./WORKFLOW.md) is responsible for ensuring that a new entry is added within atleast 2 working days.
+
+- PM has the ability to assign himself or a team member to follow up on newly created issue
+- Assigned team member must not be on holidays and should prioritize this issue
+
+## Notes for tech
+
+- A live **mainnet** instance has not been deployed yet, this means that the discribed process cannot be followed through yet.
+- Test [unreleasedTokens.spec.ts](./packages/cover-traffic/unreleasedTokens.json) validates entries on each PR, however it does not validate if the `CLAIMANT_ADDR` has claimable tokens. This is topic we will have to tackle once tokens become claimable.
+- While we may run HOPR in xDai, Goerli, or other networks, `unreleasedTokens` source of truth lives in a **mainnet** HoprDistributor contract. These tokens should only affect importance calculation when HOPR is running on `xDai` or `mainnet`, else it can be ignored.
+
+## Notes for community
+
+- User should expect his entry to be added within 1-2 working days, if it's not added, user should reach us through twitter.
```

Comment:
I propose TG instead of Twitter - approach any of the ambassadors. Todo: inform ambassadors about this process.